brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3353 Improvement: Container - Enhanced the accessibility of nested Containers.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3353

### Screenshots

- https://d.pr/v/xOtyn2

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop container.
- Select any layout, created a nested structure.
- Hover on nested container, You will see blue line on hover to access a container.

### Checklist:
